instrumentation-api/src/main/java/io/opentelemetry/instrumentation/api/instrumenter/db/SqlClientAttributesExtractorBuilder.java

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.instrumentation.api.instrumenter.db;
+
+import io.opentelemetry.api.common.AttributeKey;
+
+/** A builder of {@link SqlClientAttributesExtractor}. */
+public final class SqlClientAttributesExtractorBuilder<REQUEST, RESPONSE> {
+
+  final SqlClientAttributesGetter<REQUEST> getter;
+  AttributeKey<String> dbTableAttribute;
+
+  SqlClientAttributesExtractorBuilder(SqlClientAttributesGetter<REQUEST> getter) {
+    this.getter = getter;
+  }
+
+  /**
+   * Configures the extractor to capture the table returned by {@link
+   * SqlClientAttributesGetter#table(Object)} as span attribute.
+   *
+   * @param dbTableAttribute The {@link AttributeKey} under which the value returned by {@link
+   *     SqlClientAttributesGetter#table(Object)} will be stored.
+   */
+  public SqlClientAttributesExtractorBuilder<REQUEST, RESPONSE> captureTable(
```

Comment:
maybe rename method to `tableAttribute`? and does it make sense to default to DB_SQL_TABLE since this is "Sql"ClientAttributesExtractorBuilder?

Reply:
Good point, done.